package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "eigengraus-essential-establishment-generator",
+  "version": "2.2.1",
+  "description": "Eigengrau's Essential Establishment Generator",
+  "main": "main.txt",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator.git"
+  },
+  "author": "ryceg",
+  "license": "GNU GENERAL PUBLIC LICENSE",
+  "bugs": {
```

Comment:
should add:
```
"scripts": {
"lint": "eslint EssentialEstablishmentGenerator/**/*.js"
"lint-fix": "eslint --fix EssentialEstablishmentGenerator/**/*.js"
}
```
so that we can run `npm run lint` and `npm run lint-fix`

Reply:
Was actually just doing that, unfortunately we've still got 89 errors that can't be autofixed. 
Autofix did catch a few hundred though.